javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,11 @@
+/* exported filterOutStrings */
+
+function filterOutStrings(values) {
+  // debugger;
+  const onlyNumbers = [];
+  for (let i = 0; i < values.length; i++) {
+    if (typeof values[i] !== 'string') {
+      onlyNumbers.push(values[i]);
+    } else;
```

Comment:
If you don't plan on adding anything to your `else` statement, there's no point in including one. Just do an `if` statement.